frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -208,6 +211,10 @@ export const funnelLogic = kea<funnelLogicType>({
                 return !!(featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] && preflight?.is_clickhouse_enabled)
             },
         ],
+        funnelPersonsEnabled: [
+            () => [featureFlagLogic.selectors.featureFlags],
+            (featureFlags) => featureFlags[FEATURE_FLAGS.FUNNEL_PERSON_MODAL],
```

Comment:
Type here, it should be `FUNNEL_PERSONS_MODAL` (person+"S").

I think we should remove the explicit `Record<string, string>` type for the FEATURE_FLAGS object, but again outside the scope here :)